2023-04-30 13:22:36,657 - _internal.py:_log[line:113] - WARNING:  * Debugger is active!
2023-04-30 13:22:36,659 - _internal.py:_log[line:113] - INFO:  * Debugger PIN: 504-467-380
2023-04-30 13:22:36,678 - _internal.py:_log[line:113] - INFO:  * Running on http://localhost:1234/ (Press CTRL+C to quit)
2023-04-30 13:22:38,542 - _internal.py:_log[line:113] - INFO: 127.0.0.1 - - [30/Apr/2023 13:22:38] "[32mGET / HTTP/1.1[0m" 302 -
2023-04-30 13:22:38,556 - _internal.py:_log[line:113] - INFO: 127.0.0.1 - - [30/Apr/2023 13:22:38] "[37mGET /login/index HTTP/1.1[0m" 200 -
2023-04-30 13:23:00,879 - _internal.py:_log[line:113] - INFO:  * Detected change in 'S:\\anaconda3\\Lib\\site-packages\\sqlalchemy\\dialects\\sqlite\\__pycache__\\__init__.cpython-39.pyc', reloading
2023-04-30 13:23:00,890 - _internal.py:_log[line:113] - INFO: 127.0.0.1 - - [30/Apr/2023 13:23:00] "[32mGET /login/index?Username=root&Password=88888888 HTTP/1.1[0m" 302 -
2023-04-30 13:23:00,924 - _internal.py:_log[line:113] - INFO:  * Detected change in 'S:\\anaconda3\\Lib\\site-packages\\sqlalchemy\\dialects\\sqlite\\__pycache__\\base.cpython-39.pyc', reloading
2023-04-30 13:23:00,924 - _internal.py:_log[line:113] - INFO:  * Detected change in 'S:\\anaconda3\\Lib\\site-packages\\sqlalchemy\\dialects\\sqlite\\__pycache__\\json.cpython-39.pyc', reloading
2023-04-30 13:23:00,924 - _internal.py:_log[line:113] - INFO:  * Detected change in 'S:\\anaconda3\\Lib\\site-packages\\sqlalchemy\\dialects\\sqlite\\__pycache__\\pysqlcipher.cpython-39.pyc', reloading
2023-04-30 13:23:00,925 - _internal.py:_log[line:113] - INFO:  * Detected change in 'S:\\anaconda3\\Lib\\site-packages\\sqlalchemy\\dialects\\sqlite\\__pycache__\\pysqlite.cpython-39.pyc', reloading
2023-04-30 13:23:00,925 - _internal.py:_log[line:113] - INFO:  * Detected change in 'S:\\anaconda3\\Lib\\site-packages\\sqlalchemy\\dialects\\sqlite\\__pycache__\\dml.cpython-39.pyc', reloading
2023-04-30 13:23:00,925 - _internal.py:_log[line:113] - INFO:  * Detected change in 'S:\\anaconda3\\Lib\\site-packages\\sqlalchemy\\dialects\\sqlite\\__pycache__\\aiosqlite.cpython-39.pyc', reloading
2023-04-30 13:23:00,925 - _internal.py:_log[line:113] - INFO:  * Detected change in 'S:\\anaconda3\\Lib\\sqlite3\\__pycache__\\__init__.cpython-39.pyc', reloading
2023-04-30 13:23:00,925 - _internal.py:_log[line:113] - INFO:  * Detected change in 'S:\\anaconda3\\Lib\\sqlite3\\__pycache__\\dbapi2.cpython-39.pyc', reloading
2023-04-30 13:23:01,198 - _internal.py:_log[line:113] - INFO: 127.0.0.1 - - [30/Apr/2023 13:23:01] "[37mGET /device/index HTTP/1.1[0m" 200 -
2023-04-30 13:23:01,549 - _internal.py:_log[line:113] - INFO: 127.0.0.1 - - [30/Apr/2023 13:23:01] "[36mGET /static/device/amis/sdk.css HTTP/1.1[0m" 304 -
2023-04-30 13:23:01,550 - _internal.py:_log[line:113] - INFO: 127.0.0.1 - - [30/Apr/2023 13:23:01] "[36mGET /static/device/amis/iconfont.css HTTP/1.1[0m" 304 -
2023-04-30 13:23:01,551 - _internal.py:_log[line:113] - INFO: 127.0.0.1 - - [30/Apr/2023 13:23:01] "[36mGET /static/device/amis/helper.css HTTP/1.1[0m" 304 -
2023-04-30 13:23:01,552 - _internal.py:_log[line:113] - INFO: 127.0.0.1 - - [30/Apr/2023 13:23:01] "[36mGET /static/device/amis/sdk.js HTTP/1.1[0m" 304 -
2023-04-30 13:23:01,865 - _internal.py:_log[line:113] - INFO: 127.0.0.1 - - [30/Apr/2023 13:23:01] "[36mGET /static/device/amis/thirds/@fortawesome/fontawesome-free/webfonts/fa-solid-900.woff2 HTTP/1.1[0m" 304 -
